manifests/GitLab/Gitter.IM/5.1.0.yaml

```diff
@@ -0,0 +1,15 @@
+Id: Gitlab.Gitter.IM
+Version: 5.1.0
+Name: Gitter IM
+Publisher: Gitlab
+License: 2011-2017 GitLab B.V.
+LicenseUrl: https://gitlab.com/gitlab-org/gitter/desktop/
+AppMoniker: gitter
+Description: Great IM Client for Github
+Homepage: https://gitter.im
+Tags: "Chat, messages"
+Installers:
+  - Arch: x86
+    Url: https://update.gitter.im/win/latest
```

Comment:
BTW, instead of using "latest" url, it is wise to use direct url:

https://update.gitter.im/win/GitterSetup-5.0.1.exe

Reply:
In order to find direct url, start download with "latest" url, cancel it, copy url from browser's downlaod manager.